src/main/java/com/bunq/sdk/exception/ExceptionHandler.java

```diff
@@ -0,0 +1,58 @@
+package com.bunq.sdk.exception;
+
+import java.util.List;
+
+public class ExceptionHandler {
+
+//  HTTP error response codes constants.
+  private static final int HTTP_RESPONSE_CODE_BAD_REQUEST = 400;
+  private static final int HTTP_RESPONSE_CODE_UNAUTHORIZED = 401;
+  private static final int HTTP_RESPONSE_CODE_FORBIDDEN = 403;
+  private static final int HTTP_RESPONSE_CODE_NOT_FOUND = 404;
+  private static final int HTTP_RESPONSE_CODE_METHOD_NOT_ALLOWED = 405;
+  private static final int HTTP_RESPONSE_CODE_TOO_MANY_REQUESTS = 429;
+  private static final int HTTP_RESPONSE_CODE_INTERNAL_SERVER_ERROR = 500;
+
+//  Some glue to glue the error message together.
+  private static final String GLUE_ERROR_MESSAGES = "\n";
+
+  /**
+   *
+   * @param responseCode The HTTP response code related to the request.
+   * @param messages  The error message related to the exception that should be raised.
```

Comment:
to be "thrown" in this case; raising exceptions is the pythonic way :D

Reply:
😁